### ISSUE TITLE
hi!

### ISSUE THREAD

**Issue by Afiasarwar** [APSIMInitiative/ApsimX#4734]

**Comment by Afiasarwar** [comment]
I am running apsim to check the effect of irrigation.I used 4 irrigation treatments of same quantity that is 75 ml on different dates and in second trial i changed the amount to 65 ml but it does not effect the yield?

**Comment by Keith-Pembleton** [comment]
Can you please post your simulation and the met file so we can take a look to see what is happening
Cheers
Keith

**Comment by Afiasarwar** [comment]
Thank you for your response.I have restriction from my supervisor on sharing simulation but i can give you full overview about this problem.
we are basically running apsim for future scenerios usin RCM data from 2019 to 2099.we are interested in two types of adaptations that is sowing dates and irrigation adaptations.yield is effected by sowing date adaptation but not on irrigation.I have noticed that by keeping100% and 50% of inial water there is no effect on yield but keeping 0% of initial water c,early effect the yield so is it possible to keep intial water 0%?

**Comment by peter-devoil** [comment]
Of course. My back yard (in drought affected australia) was 0% up until a month or so ago.

The initial water is just that - a starting point. If you were able to measure the soil water content of your experiment at that time, then that's what you set it to. Anything else is an (educated) guess. After the first rainfall event it will be something else - water content is a state; and the simulation of that state has to start somewhere.

If you want to experiment further with how apsim responds to water supply; then drop the climate control (in the standard toolbox under the meteorological folder) into your simulation and reduce/increase the rainfall amount. It will be interesting to observe what happens if your environment has got extended dry spells.

**Comment by Afiasarwar** [comment]
Thank you Peter.But the problem is there is no drought in my study area.I have taken permission from my supervisor here is my simulation attached can you please figure out the problem?

[irrigation.zip](https://github.com/APSIMInitiative/ApsimX/files/4301503/irrigation.zip)

**Comment by peter-devoil** [comment]
1. in tracker, change the "to" event to harvesting, not maturity. There is no event called maturity.
2. Add another: "sum of irrigation on end_of_day from sowing to harvesting as Cum_Irrig"
3. add Cum_irrig to report variables
4. set report frequecy = "harvesting'
5. I'd suggest not using P unless there is a demonstrated need. If you do, 
5a. Add the phosporous component from standard toolbox/soil related/soil/Phosporous
5b. Configure with starting {labile/banded/rock}Phosporous
5c. in manager reseting; the name of your phophorous module is "Lyallpur Phosporous"
5d. Somehow find some wheat P parameters
...
I can't run your wheat cultivar without the modified xml file, but can give pointers:
- apsim's not going to do well with winter wheat. If that's your system we'll need help from experts.
- your simulation doesnt grow large amounts of biomass, but I wouldnt expect it to under that temperature regime, even with large amounts of water & N.
- check your phenology before you look at biomass/yield. If flowering / maturity time is off, you cant expect anything else to be "right for the right reasons". 
- check the water balance: starting water (50% of 263mm = 130mm), rain (20mm) & irrigation (4 * 55mm). What is transpiration saying?
- as you say, changing rainfall amount won't affect much. 
- check the stress tables in the summary file. 0 means no stress.

We should continue this discussion over in apsim classic.
